composer.json

```diff
@@ -20,17 +20,27 @@
         }
     ],
     "require": {
-        "craftcms/cms": "^3.1"
+        "craftcms/cms": "^3.3.0"
     },
     "repositories": [
         {
             "type": "composer",
             "url": "https://asset-packagist.org"
         }
     ],
+    "require-dev": {
+        "vlucas/phpdotenv": "^4.1.0",
+        "codeception/codeception": "^4.1.1",
+        "codeception/module-asserts": "^1.1.1",
+        "codeception/lib-asserts": "^1.1.0",
+        "codeception/module-yii2": "^1.1.0",
+        "phpunit/phpunit": "^7.5.20",
+        "codeception/module-phpbrowser": "^1.0.0"
+    },
     "autoload": {
         "psr-4": {
-          "viget\\base\\": "src/"
+            "viget\\base\\": "src/",
+            "viget\\base\\tests\\fixtures\\": "tests/fixtures/"
```

Comment:
I find this namespace's name unintuitive. Makes me expect the path `<viget\base's path>/tests/fixtures`

Reply:
Updated to `"vigetbasetests\\fixtures\\": "tests/fixtures/"` to match craft